src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
```

Comment:
The property 'Spring.demo.value' should follow the standard naming convention. It is recommended to use lowercase letters with dots as separators. For example, 'spring.demo.value'.